.circleci/readme.md

```diff
@@ -102,7 +102,7 @@ Read below for details how to generate the files manually without the `generate.
    config-2_1.yml)
    Note that if the patch no longer applies to `config-2_1.yml` a new patch file
    is needed, do this by manually making `config-2_1.yml` high resource and create
-   the patch file based on the diff (don't commit it though).
```

Comment:
I am lost... so you say the same as before - update the config-2_1.yml file and the script will just make the diffs for you?
Unfortunately running the script failed for me:
edimitri-rmbp15:.circleci ekaterina.dimitri$ ./create-patches.sh
```
Traceback (most recent call last):
  File "/Users/ekaterina.dimitri/IdeaProjects/april/cassandra/.circleci/circle-ci-config.py", line 32, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cassandra/.circleci/circle-ci-config.py", line 39, in <module>
    import yaml
ModuleNotFoundError: No module named 'yaml'
```

Reply:
Can we say what the other scripts do and when do we update them? 
Also, based on your feedback, I guess we need to add that "apply" means make the changes as per your expectations not just without error.

Reply:
@ekaterinadimitrova2 did it actually fail or did it just log that with no other output?  when that error happens we install yaml and move on

Reply:
In my experience people will complain if they have to install modules and it doesn't work our of the box. Probably one of the reasons why shell scripts were chosen